update.sh

```diff
@@ -78,7 +112,7 @@ function update_node_version() {
       sed -Ei -e "/${pattern}/d" "${dockerfile}"
     done
 
-    if [ "${variant}" = "alpine" ]; then
+    if [ "${variant}" = "alpine" ] && [ "${SKIP}" != true ]; then
       alpine_version=$(get_config "./" "alpine_version")
       sed -Ei -e "s/(alpine:)0.0/\\1${alpine_version}/" "${dockerfile}"
     fi
```

Comment:
We need an else condition that retrieves the alpine version from the image, otherwise the version would become 0.0

Reply:
Something like:

```bash
if [ "${variant}" = "alpine" ]; then
   if [ "${SKIP}" != true ]; then
     alpine_version=$(get_config "./" "alpine_version")
  else
    alpine_version=[LOGIC HERE]
  fi
  sed -Ei -e "s/(alpine:)0.0/\\1${alpine_version}/" "${dockerfile}"
fi
```

Reply:
Oh, good catch!

At some point we should maybe refactor some things out so that the yarn and alpine version updates are in their own functions.

Reply:
Yep!

Reply:
I think I have a solution, but it changes the some logic in `update_node_version` a bit